package.json

```diff
@@ -8,16 +8,16 @@
     "url": "https://github.com/collective/plone.formwidget.geolocation"
   },
   "devDependencies": {
-    "@patternslib/dev": "^3.3.5",
+    "@patternslib/dev": "^3.6.1",
     "babel-loader": "^9.1.2",
     "clean-css-cli": "^5.6.1",
     "npm-run-all": "^4.1.5",
     "sass": "^1.49.11",
     "sass-loader": "^13.2.0"
   },
   "dependencies": {
-    "@patternslib/pat-leaflet": "2.1.1",
-    "@patternslib/patternslib": "9.9.4"
+    "@patternslib/pat-leaflet": "Patternslib/pat-leaflet#housekeeping",
```

Comment:
You can change this now to: `2.2.0`.
I just released a new `pat-leaflet version.